### PR TITLE
s3same Readme updates

### PR DESCRIPTION
**Pull request by vmammanalupo**
@vokal/ios-developers  @vokal/systems 

I went through some of the CI setup processes and the area related to s3same became a bit convoluted. I took notes on some of the issues and tried to update the readme to identify some prereqs that would have saved me some time, as well as update the sudo pip install s3same command to account for the file directory error I was receiving. 

These are just suggested changes, please review and provide feedback as necessary, especially because I am not certain the extent this is used/relevant for systems folks

### CODE REVIEW

**Comment by garciajg** [comment]
@vmammanalupo @vokal-isaac do we want to merge this? I have stuff to add to the docs as well, I just don't want a conflict or repeat some info. This LGTM

**Comment by vmammanalupo** [comment]
@garciajg I was just waiting on an approval. @bryanluby any chance you can take a quick peek as well. We went through these adjusted steps together

**Review by bryanluby — APPROVED** [review]

**Pull request merged**
